plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmDatatableWidgetAction.java

```diff
@@ -133,35 +121,35 @@ protected BpmWidgetInfo storeWidgetInfo(final Widget widget) {
             final String contId = param[1];
             String sourceId = null;
             if (param.length > 2) {
-            		sourceId = param[2];
+                sourceId = param[2];
             }
             widgetInfo.setWidgetType(widget.getType().getCode());
             ApsProperties properties = new ApsProperties();
             this.setPropertiesIntoWidgetInfo(properties, procId, contId, sourceId);
             widgetInfo.setInformationDraft(properties.toXml());
             this.updateConfigWidget(widgetInfo, widget);
         } catch (Exception e) {
-            _logger.error("Error in save WidgetInfo", e);
+            logger.error("Error in save WidgetInfo", e);
 
         }
         return widgetInfo;
     }
 
     @Override
     public String chooseForm() {
+        String response = super.chooseForm();
         try {
             this.loadFieldIntoDatatableFromBpm();
-        } catch (Exception t) {
-            _logger.error("Error in chooseForm", t);
-            return FAILURE;
+        } catch (ApsSystemException t) {
+            logger.error("Error in chooseForm", t);
```

Comment:
Should this still set response = FAILURE? Seems like an important logical change

Reply:
Fixed